database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -21,7 +21,7 @@ public function up()
             $table->string('email')->unique();
             $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
-            $table->binary('avatar')->nullable();
+            $table->binary('avatar')->default('default.png');
```

Comment:
@glenbenatiro 
I have error when I run `php artisan migrate`
[![Image from Gyazo](https://i.gyazo.com/1faacc71a0c6da1b4c47f08691e1af65.png)](https://gyazo.com/1faacc71a0c6da1b4c47f08691e1af65)

I think data type should be string not binary if you will set a default value `string`.